github/github.go

```diff
@@ -3,6 +3,15 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
+// go-github authors should run the following commands before
```

Comment:
these instructions should be added to CONTRIBUTING.md.  Either in addition to, or probably instead of, here.